content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News","Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks.
```

Comment:
It seems like we are missing one more sentence in this paragraph, staging specific what it does. "Policy as Code" and "flexible rules" don't quite tell me what CrossGuard is if I am new to the overall concept.

What do you think of this, any better? (Feel free to ignore this suggestion if you disagree.)

"... or just on specific stacks. Allowing you to verify custom policies or enforcement of changes before they get applied to your resources."

Reply:
going to go w "CrossGuard allows you to verify or enforce custom policies on changes before they get applied to your resources." Wasnt really sure what was meant by "enforcement of changes "